doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,317 @@
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "name":"New Cassandra DB appender",
+    "pluginClassName":"org.kaaproject.kaa.server.appenders.cassandra.appender.CassandraLogAppender",
+    "pluginTypeName":null,
+    "tenantId":"70"
+}
+```
+
+# Playing with Cassandra log appender
+
+We'll use [Data collection demo](https://github.com/kaaproject/sample-apps/tree/master/datacollectiondemo/source) from Kaa Sendbox. Our example will send data 
+to Kaa and then persist it to Cassandra. Also, we'll do selection queries on persisted data.
+
+We have next log schema:
+
+```json
+{
+    "type":"record",
+    "name":"LogData",
+    "namespace":"org.kaaproject.kaa.schema.sample.logging",
+    "fields":[
+        {
+            "name":"level",
+            "type":{
+                "type":"enum",
+                "name":"Level",
+                "symbols":[
+                    "KAA_DEBUG",
+                    "KAA_ERROR",
+                    "KAA_FATAL",
+                    "KAA_INFO",
+                    "KAA_TRACE",
+                    "KAA_WARN"
+                ]
+            }
+        },
+        {
+            "name":"tag",
+            "type":"string"
+        },
+        {
+            "name":"message",
+            "type":"string"
+        }
+    ]
+}
+```
+
+The following JSON example matches the previous schema.
+
+```json
+{
+    "level":"KAA_INFO",
+    "tag":"TEST_TAG",
+    "message":"My simple message"
+}
+```
+
+Go to Data collection demos in Sandbox.
+
+<img src="attach/data-collection-demo-in-sandbox.png">
+
+Follow <b>Installation</b> instructions.
+
+Next, in the Admin UI follow to <b>Data collection demo</b> application
+
+<img src="attach/data-collection-demo-application.png">
+
+Go to application's <b>Log appenders</b> configuration and add a new one.
+
+<img src="attach/ log-appenders-configuration.png">
+
+Enter name of the new appender (we'll use "Cassandra")
+
+Select <b>Cassandra</b> appender type.
+
+<img src="attach/cassandra-appender-type.png">
+
+Add new node in the <b>Configuration</b> section (localhost:9042)
+
+<img src="attach/configuration-section.png">
+
+Add auth details if needed (for Sandbox it's empty)
+
+Fill keyspace name. "kaa" is used in this example, because it's already created on a Sandbox machine.
+
+"logs_example" is used as the <b>Table name pattern</b>.
+
+The important part of configuration is <b>Column Mapping</b>:
```

Comment:
Please fix the column mapping. It is useless. UUID as a primary key is not interesting. It is better to have it as a clustering key. And partition key may be a date(YYYY-MM-DD-HH) and a log type